### PR TITLE
update for sinatra security vuln

### PR DESCRIPTION
**Pull request by taquitos**
- [x] I have tested this change locally and tried to launch the server as well as access a project, and with that at least one build
- [x] If there is an existing issue, make sure to add `Fixes ...` as part of the PR body to reference the issue you're solving

Updates for security, Sinatra 2.0.3 finally plays nicely with the other gems 🎉

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -79,6 +79,11 @@ GEM
   specs:
     CFPropertyList (3.0.0)
     abstract_type (0.0.7)
+    activesupport (5.2.0)
```
Comment (KrauseFx) [inline Gemfile.lock]: As pointed out by @snatchev, this is not perfect, but not much we can do for now. I'll investigate on why this is added, and submit an issue if there isn't one already

**Review by KrauseFx — APPROVED** [review]
💯

**Comment by KrauseFx** [comment]
Alright it was added with https://github.com/sinatra/sinatra/pull/1442, but no issue where people discuss it, will create one now

**Comment by KrauseFx** [comment]
Created an issue https://github.com/sinatra/sinatra/issues/1451

**Comment by KrauseFx** [comment]
Either way, let's merge and go ahead

**Comment by taquitos** [comment]
Great, thanks for creating the issue.

**Pull request merged**